scripts/docker/data/domainindex.html

```diff
@@ -0,0 +1,43 @@
+{#
+    basic/domainindex.html
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Template for domain indices (module index, ...).
+
+    :copyright: Copyright 2007-2016 by the Sphinx team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
```

Comment:
We'll need to put this into third_party. If moving the HTML file is too difficult at least create a sphinx entry with its LICENSE.